frontend/taipy-gui/src/components/Taipy/Notification.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Alert from "@mui/material/Alert";  
+import { TaipyBaseProps } from "./utils";
+import { useDynamicProperty } from "../../utils/hooks";
+
+interface NotificationProps extends TaipyBaseProps {
+    severity?: "error" | "warning" | "info" | "success";
+    message: string | (() => string);  // Dynamic string handling
+    variant?: "filled" | "outlined" | "standard";
+    defaultMessage?: string;
+}
+
+const Notification = (props: NotificationProps) => {
+    const { severity = "info", variant = "filled" } = props;
+
+    // Use useDynamicProperty to handle dynamic message and defaultMessage
+    let displayMessage = useDynamicProperty(props.message, props.defaultMessage, undefined);
```

Comment:
use const whenever possible
Third argument should be `""` as it is passed as a children of the Alert Mui component